nginx/conf.d/1_webapp.conf

```diff
@@ -0,0 +1,47 @@
+#Modified from: https://itnext.io/docker-rails-puma-nginx-postgres-999cd8866b18
```

Comment:
About the filename: files are sorted in lexical order, not numeric order, so this will not behave the way that you think it will.

```
01_file
02_file
09_file
10_file
1_file
11_file
50_file
99_file
```

The convention is to use two digits, and if the ordering doesn't matter, use `50_` to put it in the middle.

Reply:
It looks like the other files aren't following the normal convention, so I guess it doesn't matter that much. `1_` will sort before anything that begins with a letter of the alphabet.